### ISSUE TITLE
Doc typo: `message-ttl` vs. `message_ttl`

### ISSUE THREAD

**Issue by Steve-Roderick** [MetPX/sarracenia#299]
We noticed the [documented](https://github.com/MetPX/sarracenia/blob/master/doc/sr_subscribe.1.rst#id29) `message-ttl` configuration option is not recognized (see example below) while the undocumented `message_ttl` option is recognized but not documented.  

If resolving this issue is as simple as updating the English and French docs to use `message_ttl` we are happy to submit a pull request to such effect.

Many thanks,

S.

### Reference

Using `message-ttl` option as documented results in `unknown or not declared` warning.
```
$ sr_subscribe foreground  -n sr-message-ttl-example.conf
2020-01-21 18:57:48,553 [WARNING] extended option message-ttl = ['10'] (unknown or not declared)
...
$ grep message-ttl sr-message-ttl-example.conf
queue_name q_anonymous.sr-message-ttl-example.conf
message-ttl 10
```

Using undocumented `message_ttl` option works.
```
$ sr_subscribe foreground  -n sr-message-ttl-example.conf
2020-01-21 18:59:53,264 [WARNING] message_ttl setting (10) may cause problem...too low
...
$ grep message_ttl sr-message-ttl-example.conf
message_ttl 10
```

Complete reproducible example:  
```
$ cat sr-message-ttl-example.conf
broker amqps://anonymous@dd.weather.gc.ca
exchange xpublic
queue_name q_anonymous.sr-message-ttl-example.conf

subtopic *.radar.CAPPI.GIF.CASBE/*#
directory /tmp/data
instances 1
message-ttl 10
```

**Comment by petersilva** [comment]
Thanks for the excellent report!  correcting documentation to indicate message_ttl as the correct name.
next release (not yet planned) will have the fix.

**Comment by petersilva** [comment]
released in 2.20.01b4
